irods/test/scripts/test002_write_native_credentials_to_secrets_file.bats

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bats
 #
-# Test creation of .irodsA for iRODS native authentication using the free function,
-#    irods.client_init.write_pam_credentials_to_secrets_file
+# Test creation of .irodsA for iRODS pam_password authentication using the free function,
+#    irods.client_init.write_native_credentials_to_secrets_file
```

Comment:
I don't see what this file has to do with pam_password authentication... Am I just missing something?

Reply:
Several things going in under one PR. All Pam related through 1 or 2 degrees of Kevin Bacon

Reply:
Well, what I mean is... PAM is no longer referenced at all in this specific file. It seems exclusively related to native iRODS authentication. Just wondering whether we should replace the word "pam_password" with "native" to prevent future confusion?

Reply:
got it  ... those two heading comments are still reversed.  Correcting.